FFXIVClientStructs/FFXIV/Client/UI/Misc/RaptureTextModule.cs

```diff
@@ -54,17 +54,106 @@ public unsafe partial struct RaptureTextModule {
+    public partial CStringPointer FormatAddonText2StringIntUIntUIntUIntUInt(uint addonId, CStringPointer strParam, int intParam, uint uintParam1, uint uintParam2, uint uintParam3, uint uintParam4);
+
+    [MemberFunction("E8 ?? ?? ?? ?? 48 8B 8C 24 ?? ?? ?? ?? 45 33 C9 4C 8B C0 C6 44 24 ?? ??")]
+    public partial CStringPointer FormatAddonText2StringIntUIntUIntUIntUIntUInt(uint addonId, CStringPointer strParam, int intParam, uint uintParam1, uint uintParam2, uint uintParam3, uint uintParam4, uint uintParam5);
 
-    [MemberFunction("E8 ?? ?? ?? ?? EB 51 0F B6 DB")] // FormatAddonText2<int,int>
+    [MemberFunction("E8 ?? ?? ?? ?? 48 8B D0 48 8B 8F ?? ?? ?? ??")]
+    public partial CStringPointer FormatAddonText2StringString(uint addonId, CStringPointer strParam1, CStringPointer strParam2);
+
+    [MemberFunction("40 53 55 56 57 41 54 41 56 41 57 48 83 EC 50 48 8B 05 ?? ?? ?? ?? 48 33 C4 48 89 44 24 ?? 4C 8B BC 24 ?? ?? ?? ?? 48 8B F9 48 81 C1 ?? ?? ?? ?? 4D 8B F1 49 8B D8 44 8B E2 E8 ?? ?? ?? ?? 48 85 DB 48 8D 15 ?? ?? ?? ?? 48 8D 8F ?? ?? ?? ?? 48 0F 45 D3 E8 ?? ?? ?? ?? 33 D2 45 33 C0 8D 4A 70 E8 ?? ?? ?? ?? 48 8B D8 48 85 C0 74 18 48 8D 97 ?? ?? ?? ?? 48 8B C8 E8 ?? ?? ?? ?? C7 43 ?? ?? ?? ?? ?? EB 02 33 DB 48 8D 44 24 ?? 48 89 5C 24 ?? 48 8D 54 24 ?? 48 89 44 24 ?? 48 8D 8F ?? ?? ?? ?? C6 44 24 ?? ?? E8 ?? ?? ?? ?? 48 8B 5C 24 ?? 48 85 DB 74 28 80 7C 24 ?? ?? 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 4D 85 F6 48 8D 15 ?? ?? ?? ?? 48 8D 8F ?? ?? ?? ?? 49 0F 45 D6 E8 ?? ?? ?? ?? 33 D2 45 33 C0 8D 4A 70 E8 ?? ?? ?? ?? 48 8B D8 48 85 C0 74 18 48 8D 97 ?? ?? ?? ?? 48 8B C8 E8 ?? ?? ?? ?? C7 43 ?? ?? ?? ?? ?? EB 02 33 DB 48 8D 44 24 ?? 48 89 5C 24 ?? 48 8D 54 24 ?? 48 89 44 24 ?? 48 8D 8F ?? ?? ?? ?? C6 44 24 ?? ?? E8 ?? ?? ?? ?? 48 8B 5C 24 ?? 48 85 DB 74 28 80 7C 24 ?? ?? 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 4D 85 FF 48 8D 15 ?? ?? ?? ?? 48 8D 8F ?? ?? ?? ?? 49 0F 45 D7 E8 ?? ?? ?? ?? 33 D2 45 33 C0 8D 4A 70 E8 ?? ?? ?? ?? 48 8B D8 48 85 C0 74 18 48 8D 97 ?? ?? ?? ?? 48 8B C8 E8 ?? ?? ?? ?? C7 43 ?? ?? ?? ?? ?? EB 02 33 DB 48 8D 44 24 ?? 48 89 5C 24 ?? 48 8D 54 24 ?? 48 89 44 24 ?? 48 8D 8F ?? ?? ?? ?? C6 44 24 ?? ?? E8 ?? ?? ?? ?? 48 8B 5C 24 ?? 48 85 DB 74 28 80 7C 24 ?? ?? 75 21 48 8B 1B FF 4B 68 83 7B 68 00 7F 15 48 8B CB E8 ?? ?? ?? ?? BA ?? ?? ?? ?? 48 8B CB E8 ?? ?? ?? ?? 4C 8D 87 ?? ?? ?? ??")]
+    public partial CStringPointer FormatAddonText2StringStringString(uint addonId, CStringPointer strParam1, CStringPointer strParam2, CStringPointer strParam3);
```

Comment:
```suggestion
```
No xrefs and a giant signature that detects multiple functions when scanning in IDA